pkg/ebpf/netlink/conntracker.go

```diff
@@ -281,29 +288,33 @@ func (ctr *realConntracker) unregister(c ct.Con) int {
 		return 0
 	}
 
-	key, ok := formatKey(c)
-	if !ok {
-		return 0
-	}
+	unregisterTuple := func(keyTuple *ct.IPTuple) {
```

Comment:
Is there a reason this is a closure instead of a function off `ctr` ?

Reply:
not really, I added as an internal function because I though it wouldn't make sense to "expose" this function individually. but since `register` is already private I'm happy to change it if you feel agains that.